modules/tinymce/src/core/main/ts/textpatterns/utils/Utils.ts

```diff
@@ -38,17 +38,23 @@ const deleteRng = (dom: DOMUtils, rng: Range, isRoot: (e: Node) => boolean, clea
 const getParentBlock = (editor: Editor, rng: Range): Optional<Element> =>
   Optional.from(editor.dom.getParent(rng.startContainer, editor.dom.isBlock));
 
+const sortPatterns = <T extends Pattern>(patterns: T[]): T[] => Arr.sort(patterns, (a, b) => {
```

Comment:
This logic is specific to block patterns so this name/location may not make sense.